tests/test_api.py

```diff
@@ -228,6 +228,20 @@ def test_mode_all(self):
             full_path = os.path.join(FILE_DIR, "perf_files", "api", "test_api.perf.yml")
             assert os.path.exists(full_path)
 
+    @override_settings(PERF_REC={"MODE": "overwrite"})
+    def test_mode_overwrite(self):
```

Comment:
Not sure this is the best test; I guess this would ideally validate something about the file having actually been updated?

Reply:
53056726e9d4f4ec9db8ff479b3412431008b2fd